src/components/HomePage/UpcomingEvents.js

```diff
@@ -10,9 +10,10 @@ import workshopSchedule from '../../data/WorkshopSchedule';
 import Event from './Event';
 
 // When changing this, make sure to update src/components/SchedulePage/Schedule.js as well.
-const scheduleAvailable = false;
+const scheduleAvailable = true;
```

Comment:
Can you change line "Upcoming Workshops" on line 87 of this file to say "Upcoming Events"?